client/app/src/accounts/AccountService.js

```diff
@@ -282,6 +282,19 @@
       });
       return deferred.promise;
     };
+    
+    function getAllRegisteredDelegates() {
+      var deferred = $q.defer();
+      networkService.getFromPeer("/api/delegates").then(function (resp) {
```

Comment:
Watch out it gives only the 51 active delegates, so it would be impossible to vote for other delegates